Gruntfile.js

```diff
@@ -77,7 +95,21 @@ module.exports = function(grunt){
         options: {
           spawn: false,
         }
-      }
+      },
+			js: {
```

Comment:
the `js` and the `images` watch tasks can just be removed and reverted to what was there before https://github.com/alphagov/govuk_prototype_kit/blob/master/Gruntfile.js#L79-L83